### PR TITLE
CLIP-1686: Release version 1.7.0

### PR DESCRIPTION
**Pull request by tarka**
## Pull request description

_Provide description for the PR_

## Checklist
- [ ] I have added unit tests
- [x] I have applied the change to all applicable products
- [x] I have added the change description to the `changelog.md` and `Chart.yaml` files
- [ ] (Atlassian only) I have run the E2E test (if applicable)
- [ ] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: src/main/charts/bamboo-agent/Changelog.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## 1.7.0
+
+**Release date:** 2022-10-24
```
Comment (nghazali) [inline src/main/charts/bamboo-agent/Changelog.md:5]: We need to update this date based on the time of merge

File: src/main/charts/bamboo-agent/Changelog.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+
+![AppVersion: 9.0.0](https://img.shields.io/static/v1?label=AppVersion&message=9.0.0&color=success&logo=)
+![Kubernetes: >=1.19.x-0](https://img.shields.io/static/v1?label=Kubernetes&message=>=1.19.x-0&color=informational&logo=kubernetes)
+![Helm: v3](https://img.shields.io/static/v1?label=Helm&message=v3&color=informational&logo=helm)
+
+* Pin Python version to 3.9.14 (#468)
+* Update Bitbucket ingress testing values (#466)
+* Ingress Class Name is moved under spec field. (#451)
+* CLIP-1520: Make ATL_FORCE_CFG_UPDATE configurable in values.yaml (#454)
```
Comment (nghazali) [inline src/main/charts/bamboo-agent/Changelog.md:14]: Please remove ticket number from description . As far as we have the PR# then we can trace it and no need to expose much info here.

File: src/main/charts/bamboo-agent/Chart.yaml [hunk 3]
```diff
@@ -21,11 +21,40 @@ annotations:
...
+      links:
+        - name: Github PR
+          url: https://github.com/atlassian/data-center-helm-charts/pull/466
+    - kind: changed
+      description: Ingress Class Name is moved under spec field.
+      links:
+        - name: Github PR
+          url: https://github.com/atlassian/data-center-helm-charts/pull/451
+    - kind: changed
+      description: CLIP-1520: Make ATL_FORCE_CFG_UPDATE configurable
```
Comment (nghazali) [inline src/main/charts/bamboo-agent/Chart.yaml:39]: The same here

File: src/main/charts/bamboo-agent/Changelog.md [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
+* Add osquery related env vars for Terraform (#462)
+* AWS cleanup and log colletion from k8s (#461)
+* Update the index for release 1.6.0
```
Comment (nghazali) [inline src/main/charts/bamboo-agent/Changelog.md:17]: This is not functional change and no need to list here.

**Review by nghazali — CHANGES_REQUESTED** [review]

File: src/main/charts/bamboo-agent/Changelog.md [hunk 5]
```diff
@@ -1,5 +1,21 @@
 # Change Log
 
+## 1.7.0
```
Comment (nghazali) [inline src/main/charts/bamboo-agent/Changelog.md:3]: Do we added a new feature in this release? I see adding some env variable for os query but it does not looks a new feature to me. In this case we may pick `1.6.1` for the release version instead. WDYT?

Comment (tarka) [reply src/main/charts/bamboo-agent/Changelog.md:3]: The `forceConfigUpdate` flag is new to the charts, so bumping the minor version seems appropriate.

**Review by bianchi2 — APPROVED** [review]

**Pull request merged**